src/utils/files/uploadSchemaFile.ts

```diff
@@ -0,0 +1,73 @@
+import fs from 'fs';
+import config from 'config';
+import i18next from 'i18next';
+import { logger } from '@services/logger.service';
+import { BlobServiceClient } from '@azure/storage-blob';
+import { printSchema } from 'graphql';
+import buildSchema from '@utils/schema/buildSchema';
+
+/** Storage connection string */
+// const STORAGE_CONNECTION_STRING: string = config.get('public.fileName');
+const STORAGE_CONNECTION_STRING: string = config.get(
+  'blobStorage.connectionString'
+);
+
+/**
+ * Generate and save GraphQL schema file.
+ */
+const generateAndSaveSchema = async () => {
+  // GraphQL schema object
+  const schema = await buildSchema();
+  // Print schema to string
+  const schemaString = printSchema(schema);
+  // Save schema to file and return it
+  fs.writeFileSync('schema.graphql', schemaString);
+  return fs.readFileSync('schema.graphql');
+};
+
+/**
+ *  Upload GraphQL schema file in public storage.
+ *
+ * @param containerName main container name
+ * @param blobName blob name
+ * @param file GraphQL schema file
+ */
+const uploadFile = async (
+  containerName: string,
+  blobName: string,
+  file: any
+) => {
+  try {
+    const blobServiceClient = BlobServiceClient.fromConnectionString(
```

Comment:
@AntoineRelief I can't upload the created schema file into the storage enpoint, I always getting this error:
![image](https://github.com/ReliefApplications/ems-backend/assets/28535394/9e7da5a6-9978-4099-a324-ebf2543e0f5e)